dspace-api/src/main/java/org/dspace/util/SolrImportExport.java

```diff
@@ -298,8 +299,11 @@ private static void reindex(String indexName, String exportDirName, boolean keep
             }
 
             // Create a temp directory to store temporary core data
-            File tempDataDir = new File(configurationService.getProperty(
-                "dspace.dir") + File.separator + "temp" + File.separator + "solr-data");
+            File tempDataDir = new File(configurationService.getProperty("solr.home") + File.separator + "server"
+                               + File.separator + "solr"
+                               + File.separator + "configsets" + File.separator + "statistics"
```

Comment:
We shouldn't hardcode `statistics` here and instead use `instanceIndexName`.  The index might have a different name.